tools/lib/logreader.py

```diff
@@ -118,6 +119,8 @@ def __iter__(self):
       else:
         yield ent
 
+IterableLog = Union[LogReader, List[capnp._DynamicStructReader]]
```

Comment:
typing LogReader's `__iter__` might be a good idea too `Iterator[capnp._DynamicStructReader]`. Then you can just type `Iterable[capnp._DynamicStructReader]` without this custom type var?

Reply:
typed logreader and multilogiterator with iterator, but I think the custom type is still useful so you don't have to type out

```Iterable[capnp._DynamicStructReader]```

every time